composer.json

```diff
@@ -12,7 +12,8 @@
         "issues": "https://github.com/PHLAK/Twine/issues"
     },
     "require": {
-        "php": ">=7.0"
+        "php": ">=7.0",
+        "ext-openssl": "*"
```

Comment:
When I test the `Encryptable` trait, I notice that it uses the `openssl_*` functions to do the encrypt/decrypt work.

I think we need to add this line inside `require` block to check whether the `openssl` extension is installed.

Reply:
This is good, thank you.